packages/freshmint/commands/burn.ts

```diff
@@ -0,0 +1,35 @@
+import { Command } from 'commander';
+import ora from 'ora';
+import chalk from 'chalk';
+
+import { FlowGateway, FlowNetwork } from '../flow';
+import { FreshmintError } from '../errors';
+import { loadConfig } from '../config';
+import { parseUFix64Argument } from '../arguments';
+
+export default new Command('burn')
```

Comment:
In addition to defining the `burn` command here, you'll need to add it to the main program in `freshmint/index.ts`. You'll see the other commands there. Probably makes sense to add it after the `mint` command since burn is the complement of mint.